docs/manual/upgrades.md

```diff
@@ -75,6 +75,15 @@ This reconciler should ensure that the cluster is moving faster towards the desi
 
 Once all Pods are updated to the new image the upgrade is done and the cluster status of the FoundationDB cluster resource in Kubernetes should show that the reconciliation is done.
 
+#### Note on Recreation vs Replacement
+
+Do note that both "recreation" and "replacement" have a specific meaning here.  Recreation of a pod specifically means that a new \
```

Comment:
Do you think it's enough to link this doc: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/docs/manual/replacements_and_deletions.md#replacements-and-deletions here? That doc should cover everything.

Reply:
I think if you sub deletion for recreation it does, but I didn't get the equivalence of the term deletion to recreation with this doc, I'm thinking this might do it though:
`Do note that both "recreation" (also referred to as "deletion" of a pod/process group) and "replacement" have a specific meaning here, see https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/docs/manual/replacements_and_deletions.md#replacements-and-deletions for more information`

Reply:
That sounds good 👍